src/components/SafeExitButton.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Flex, Button } from "@radix-ui/themes";
+import Image from "next/image";
+
+const handleClick = () => {
+	window.open("https://www.google.com", "_self", "noopener,noreferrer");
+};
```

Comment:
acho que o `replace` é melhor, pq ele apaga o site atual do historico

tb sugiro da gente tentar excluir o dominio `mapadoacolhimento.org` do historico. ve se isso ajuda -> https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/history/deleteUrl

se nao for possivel, tudo bem, o replace já é uma boa

Reply:
Consegui utilizar o replace, mas a função deleteUrl não :/